### PR TITLE
Update GOVERNANCE.md

### PR DESCRIPTION
**Pull request by jasuarez90**
Updated instructions for onboarding regular members. (Note: This is my first time doing a pull request)

### CODE REVIEW

**Review by joesepi — APPROVED** [review]

**Comment by joesepi** [comment]
I wonder if this sort of process work should even be in the GOVERNANCE.md file.

**Review by ljharb — APPROVED** [review]
probably not, but it should be moved in a separate PR

File: governance/GOVERNANCE.md [hunk 1]
```diff
@@ -156,8 +156,9 @@ In addition to being an [Active OpenJS Collaborator][], the PR to add a Regular
 Once a PR is ready to be landed, the CPC member who lands the pull request should:
 
 * Send a notification to the project contacts for the project identified in the PR indicating that a new Regular CPC member has joined the CPC on behalf of the project.
-* Add the member to the github `cpc-regular-members` [team][cpc regular members team]
-* Add the member to the `cpc-private` email list and private directory by opening a PR against the [OpenJS Foundation CPC directory][]
+* Contact OpenJS Foundation Operations <operations@openjsf.org> and let them know we have a new Regular CPC Member:
+   * Operations will add the member to the github `cpc-regular-members` [team][cpc regular members team]
+   * Operartions will add the member to the `cpc-private` email list
```
Comment (ljharb) [inline governance/GOVERNANCE.md:161]:
```suggestion
   * Operations will add the member to the `cpc-private` email list
```

**Review by ljharb — CHANGES_REQUESTED** [review]
`[OpenJS Foundation CPC directory][]` is now unused, is that intentional? If so, please remove the definition.

**Review by ljharb — APPROVED** [review]

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by anonrig — APPROVED** [review]

**Pull request merged**